src/clom/command.py

```diff
@@ -354,13 +358,22 @@ def __init__(self, clom, name, parent=None):
         # Arguments to pass to the command line
         self._args = []
 
+    def __sort_kwargs(self, kwargs):
```

Comment:
Agreed that this is a bit awkward but it works. Maybe naming it `_parse_kwargs`?